### DISCUSSION TITLE
Is it possible to use context inside react-pdf components?

### DISCUSSION THREAD

**Discussion by joseDaKing** [diegomura/react-pdf#1593] [Q&A] [answered]
Is it possible to use context inside react-pdf components?

**Comment by jeetiss** [comment, marked as answer]
@joseDaKing sure!
hooks and context work fine with `react-pdf` components, but you can't share the context between `react-dom` and `react-pdf`, because `react-pdf` creates new reconciler that doesn't know anything about parent context and react doesn't provide a way to hookup contexts automatically. You can check workaround in this issue https://github.com/diegomura/react-pdf/issues/743
